### DISCUSSION TITLE
Git Backup and Sync Targets not working for Wiki.js

### DISCUSSION THREAD

**Discussion by RajaATAli** [requarks/wiki#7214] [Help / Questions]
**[Everything I did according to the documentation](https://docs.requarks.io/storage/git)**
1. Generating new key
```
@RajaATAli ➜ /workspaces/repo_name (main) $ ssh-keygen -t rsa -b 4096
Generating public/private rsa key pair.
Enter file in which to save the key (/home/codespace/.ssh/id_rsa): /etc/wiki/github.pem
Enter passphrase (empty for no passphrase): 
Enter same passphrase again: 
Your identification has been saved in /etc/wiki/github.pem
Your public key has been saved in /etc/wiki/github.pem.pub
The key fingerprint is:
SHA256: ....... ...... codespace@codespaces-faec29
The key's randomart image is:
+---[RSA 4096]----+
|     +*o         |
|    oo .         |
.............
|+oo=*o  .o       |
+----[SHA256]-----+
@RajaATAli ➜ /workspaces/repo_name(main) $ cd /etc/wiki
@RajaATAli ➜ /etc/wiki $ ls
github.pem  github.pem.pub
@RajaATAli ➜ /etc/wiki $ vim github.pem.pub 
@RajaATAli ➜ /etc/wiki $ vim github.pem
@RajaATAli ➜ /etc/wiki $ vim github.pem
```

2. I then added the public key `github.pem.pub` to the GitHub Repository (which is part of an internal organization so it is not public) with write access
![Screenshot 2024-06-08 at 10 44 20 PM](https://github.com/requarks/wiki/assets/76918821/5fce47fa-c35d-44ba-b6ea-9e9446cf0714)

3. This is my configuration on wiki.js
![Screenshot 2024-06-08 at 10 48 05 PM](https://github.com/requarks/wiki/assets/76918821/67a0caaa-d6fe-4970-85bc-02cc16ff0048)
![Screenshot 2024-06-08 at 10 49 27 PM](https://github.com/requarks/wiki/assets/76918821/cb9adaa2-c236-4d00-9e41-dca733cedb25)
![Screenshot 2024-06-08 at 10 50 33 PM](https://github.com/requarks/wiki/assets/76918821/4e85903a-103c-4fa2-8746-65860a15cb13)
0aa45fb)
![Screenshot 2024-06-08 at 10 50 44 PM](https://github.com/requarks/wiki/assets/76918821/9b84db18-7516-49a4-ae3e-d5f3516d961e)

4. The sync and backup still fails
![Screenshot 2024-06-08 at 10 51 28 PM](https://github.com/requarks/wiki/assets/76918821/ac5fd70f-4f5a-43a4-b208-62f9aa40a7fa)

5. This is my system info
![Screenshot 2024-06-08 at 10 56 32 PM](https://github.com/requarks/wiki/assets/76918821/04e300b2-3138-450d-8123-44a9a436150c)

**Comment by RajaATAli** [comment]
@NGPixel, Is it possible you could help with this. Thank you :)

**Comment by RajaATAli** [comment]
So it is working now
![Screenshot 2024-06-10 at 10 06 50 AM](https://github.com/requarks/wiki/assets/76918821/fc46a059-d8bd-4e71-b6fa-7a9b0891e485)

However, I can't seem to add untracked changes as I get the following error:
![Screenshot 2024-06-10 at 10 06 36 AM](https://github.com/requarks/wiki/assets/76918821/f8634924-ce1f-4565-8cb8-9dce5852c83d)

**Reply by Techeuroadmin** [reply]
What is the problem? because i have the shame error and i don't know how to resolve it
